pylint/checkers/base/basic_checker.py

```diff
@@ -168,7 +168,8 @@ class BasicChecker(_BasicChecker):
             "exec-used",
             'Used when you use the "exec" statement (function for Python '
             "3), to discourage its usage. That doesn't "
```

Comment:
```suggestion
            'Raised when the "exec" statement is used. It's dangerous to use this function'
            " for a user input and it's also slower than actual code in general. This doesn't "
            " mean you should never use it, but you should consider alternatives first."
```

Reply:
Yes, looks better. The `basic_checker.py` has been updated with your text.